lib/core/src/lib/auth/services/user-access.service.spec.ts

```diff
@@ -36,6 +36,12 @@ describe('UserAccessService', () => {
         appConfigService = TestBed.inject(AppConfigService);
     });
 
+    /**
```

Comment:
I'm wondering if we need documentation for functions in spec files as they are mostly used by tests for example to reduce repeated code and won't be used by external people like in public feature code. But if we want documentation for test functions then it's fine to me.

Reply:
just following the default docs setup for the eslint, we can later improve the configuration

Reply:
Ok @DenysVuika fine. For future we can maybe try something similarly like we have in ACA: 

<img width="873" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/06ffe3ec-847a-4c7b-8bc9-755401ef82b3">


You can see there is excludeFields field so it allows to exclude some files from specific rule.